bescort.lic

```diff
@@ -1394,7 +1394,12 @@ class Bescort
     if Flags['pattern-shift']
       Flags.reset('pattern-shift')
     else
+      Flags.reset('walking-circles')
       move Regexp.last_match[:direction]
```

Comment:
Feedback from [VTCifer](https://discord.com/channels/745675889622384681/745676101392793651/829707485908434995) in Lich discord:

> I think you're still going to run into an issue with missing match in move unless you define a new move method specific to dr-scripts instead of core lich.